feat: add skeleton for browse depositions page

kne42 opened the pull request:
#773

![Screenshot 2024-06-21 at 1 23 55 PM](https://github.com/chanzuckerberg/cryoet-data-portal/assets/29165011/fc7f2e73-5d06-4297-80fe-38c6c042ac86)

kne42 commented:
https://dev-kira-1.cryoet.dev.si.czi.technology/browse-data/depositions

note: for some reason switching between tabs takes longer in dev link vs locally

Janeece reviewed (changes requested):
@kne42 wow, i can't believe it's already this far! 
Note: in my review i'm assuming data & images in the depositions table, including what are listed depositions, is all placeholder - will not comment on this

Feedback:
- [x] (1) Nice to Have - while we are working on this, can we fix the spacing for both tabs to be accurate to the design? The space between the tab label and the underline is too big. 
    - Note: if this is better to fix in the SDS refactor, then ignore
    - <img width="527" alt="Screenshot 2024-06-24 at 10 29 04 AM" src="https://github.com/chanzuckerberg/cryoet-data-portal/assets/63013360/5c32a600-2e45-41d0-af0a-07d1af5640db">

- [ ] (2) Should have, not blocker - space between keyphoto and deposition name shouldn't expand (keyphoto width should be fixed size). 
    - If this is how other tables behave it should be fixed across all, but its a low priority. 
    - if this proves difficult to control/correct, we can prioritize other work first. 
    - <img width="565" alt="Screenshot 2024-06-24 at 10 32 00 AM" src="https://github.com/chanzuckerberg/cryoet-data-portal/assets/63013360/00e0a7f6-3d63-4f3d-a855-03cbda5863a7">


- [ ] (3) Should have, not blocker - There is a tooltip when hovering over the date that repeats the date - please remove 
    - <img width="163" alt="Screenshot 2024-06-24 at 10 20 27 AM" src="https://github.com/chanzuckerberg/cryoet-data-portal/assets/63013360/921a1aeb-9955-42b6-87ca-0e7606446a2f">

- [ ] (4) Should have, not blocker - object shape column wraps overflow, should hide overflow instead
    - <img width="624" alt="Screenshot 2024-06-24 at 10 22 53 AM" src="https://github.com/chanzuckerberg/cryoet-data-portal/assets/63013360/5a8683c4-a16c-4dc0-86e9-c1634151869d">

kne42 commented:
> * [ ]  (1) Nice to Have - while we are working on this, can we fix the spacing for both tabs to be accurate to the design? The space between the tab label and the underline is too big.
fixed

> * [ ]  (2) Should have, not blocker - space between keyphoto and deposition name shouldn't expand (keyphoto width should be fixed size).
>   
>   * If this is how other tables behave it should be fixed across all, but its a low priority.
>   * if this proves difficult to control/correct, we can prioritize other work first.

both of your bullets are true, so can't fix rn

> * [ ]  (3) Should have, not blocker - There is a tooltip when hovering over the date that repeats the date - please remove

fixed

> * [ ]  (4) Should have, not blocker - object shape column wraps overflow, should hide overflow instead

half-fixed – no longer wraps; however wont do ellipsis atm since the entire table scrolls when overflowing – do you have the link for the exact specs? i didnt see them in the linked figma

Janeece commented:
> > * [ ]   (2) Should have, not blocker - space between keyphoto and deposition name shouldn't expand (keyphoto width should be fixed size).
> >   
> >   * If this is how other tables behave it should be fixed across all, but its a low priority.
> >   * if this proves difficult to control/correct, we can prioritize other work first.
> 
> both of your bullets are true, so can't fix rn

No problem, thanks for letting me know!

> > * [ ]   (4) Should have, not blocker - object shape column wraps overflow, should hide overflow instead
> 
> half-fixed – no longer wraps; however wont do ellipsis atm since the entire table scrolls when overflowing – do you have the link for the exact specs? i didnt see them in the linked figma

@kne42  The horizontal scroll works fine as a solution, it solves the main issue of avoiding text wrap on these lists.  
The spec ([here](https://www.figma.com/design/Rrr54QysCLa14vOnQcmejb/Janeece---Annotation-Depositions?m=dev&node-id=3227-62513)) has a min/max width recommendations but I know those aren't simple to implement exactly. It's not a blocker, so lmk if we won't be doing it & I'll approve the PR.

kne42 commented:
I've added the min/max widths but am keeping the scrolling for now since the object shape type column won't seem to size to its min no matter what I do

Janeece reviewed (approved):
@kne42 thanks for making the changes! looks great!

kne42 commented on frontend/packages/data-portal/app/components/Tabs.tsx:
```diff
@@ -20,32 +20,34 @@ export function Tabs<T extends string>({
-        />
-      ))}
-    </MUITabs>
+    <div className="flex items-center">
```
couldnt find another way to centre it; css is finnicky. the div keeps the parent flex from force-stretching our tabs which is the reason behind why the offset on the indicator was so large
